specs/binding-tcp.spec/src/main/scripts/io/aklivity/zilla/specs/binding/tcp/schema/tcp.schema.patch.json

```diff
@@ -163,6 +171,8 @@
         "path": "/$defs/options/binding/tcp",
         "value":
         {
+            "title": "tcp Options",
```

Comment:
I don't think we've been adding `title` for other options, and if we do then it's just `Options` right, since it is contextual within the current scope.